doc/source/conf.py

```diff
@@ -80,3 +83,120 @@
 
 # The master toctree document.
 master_doc = "index"
+
+# change the preamble of latex with customized title page
+latex_elements = {
+    "sphinxsetup": "",
+    "passoptionstopackages": r"""
+    \PassOptionsToPackage{table}{xcolor}""",
+    "preamble": r"""\renewcommand{\sphinxmaketitle}{%
+    \newsavebox\imagebox
+    \newsavebox\myTitle
+    % native LaTeX command. Specifies a page with no formatting.
+    % Page number restarts after this page
+    \begin{titlepage}
+    % san serif
+    \sffamily
+    %
+    % set to 0 so that we don't have any blank space at the top of the page
+    \topskip0pt
+    %
+    % removes all margins
+    \advance\voffset -1in %
+    \advance\hoffset -1in %
+    \advance\vsize 2in %
+    \advance\hsize 2in %
+    %
+    % load the graphics into a LaTeX (not TeX) "savebox". We use this load
+    % to obtain the height later, and to insert it into the doc later
+    \sbox{\imagebox}{\includegraphics{../../drawing.pdf}}%
+    %
+    % put the picture in a zero sized box to get a "background effect"
+    % everything else will draw over it
+    \hbox to 0in {\vbox to 0in {\usebox{\imagebox}}}
+    %
+    % how it worked before savebox method
+    %\hbox to 0in {\vbox to 0in {\includegraphics{../../drawing.pdf}}} %
+    % Where the right boundary of the title block goes
+    %
+    % Define "ansysBlockWidth" <-- exactly what its name implies
+    \newdimen\ansysBlockWidth\ansysBlockWidth = 3in %
+    % Define "ansysBlockDistanceFromRightEdge" <-- exactly what its name implies
+    \newdimen\ansysBlockDistanceFromRightEdge \ansysBlockDistanceFromRightEdge = 0.5in %
+    % Define "distanceToBottomOfAnsysBox" <-- length from top of page to bottom of title box
+    \newdimen\distanceToBottomOfAnsysBox %
+    \distanceToBottomOfAnsysBox=\ht\imagebox %
+    \advance\distanceToBottomOfAnsysBox -\lineskip %
+    \advance\distanceToBottomOfAnsysBox -0.125in %
+    % Define "myHBoxWidth" <-- distance from left edge of page to right edge Ansys box
+    \newdimen\myHBoxWidth \myHBoxWidth=\hsize \advance\myHBoxWidth-\ansysBlockDistanceFromRightEdge
+    % implement the Ansys block
+    \hbox to \myHBoxWidth {\vbox to \distanceToBottomOfAnsysBox {}  %
+        \hfil
+        %
+        % draw the box
+        \parbox[b]{\ansysBlockWidth}{\raggedleft {
+            \includegraphics{../../logo-1.pdf}} \\
+            \footnotesize
+            \textcolor{lightgray}
+            \hrule
+            \vskip 6pt
+            \textcopyright{} 2022 ANSYS, Inc. or affiliated companies\\
+            Unauthorized use, distribution, or duplication prohibited.}
+    } %
+    %we use a save box method so that we can get draw the bottom material with respect
+    %to the bottom of the page
+    \sbox{\myTitle}{\vbox{
+    %vskip skips down a specified distance from the previous Ansys block and picture
+    \vskip 1in
+    %title
+    \centerline{\huge \bfseries PyAnsys Developer's Guide}
```

Comment:
This seems to be the only parameter users can control, that is the title. We must look for a way of reusing either the `project` variable, declared at the very beginning of the `conf.py`, or provide some variable.

This way users can forget about LaTeX.

Reply:
Unlike [[the HTML builders](https://www.sphinx-doc.org/en/master/usage/theming.html#html-themes)], the latex builder does not benefit from prepared themes. The [Options for LaTeX output(https://www.sphinx-doc.org/en/master/usage/configuration.html#latex-options)], provides much of the interface for customization